### ISSUE TITLE
playgroundEnabled not working in configuration (Docker setup)

### ISSUE THREAD

**Issue by pitininja** [librecaptcha/lc-core#175]
Hello,

I've setup LibreCaptcha with Docker on a server. I use this docker-compose:

```yaml
version: '3'

services:
  librecaptcha:
    container_name: librecaptcha
    image: librecaptcha/lc-core:latest
    restart: always
    volumes: 
      - librecaptcha_data:/lc-core/data
    networks:
      - librecaptcha-network
    ports: 
      - 127.0.0.1:8888:8888

volumes:
  librecaptcha_data:

networks:
  librecaptcha-network:
    name: librecaptcha
```

I've edited the configuration file located in the Docker volume with this configuration:

```json
{
    "randomSeed": 1234,
    "port": 8888,
    "playgroundEnabled": false,
    "throttle": 10,
    "captchaExpiryTimeLimit": 5,
    "bufferCount": 1000,
    "threadDelay": 2,
    "corsHeader": "",
    "maxAttemptsRatio": 0.01,
    "captchas": [{
        "name": "FilterChallenge",
        "allowedLevels": [
            "medium",
            "hard"
        ],
        "allowedMedia": [
            "image/png"
        ],
        "allowedInputType": [
            "text"
        ],
        "config": {}
    }, {
        "name": "GifCaptcha",
        "allowedLevels": [
            "hard"
        ],
        "allowedMedia": [
            "image/gif"
        ],
        "allowedInputType": [
            "text"
        ],
        "config": {}
    }, {
        "name": "ShadowTextCaptcha",
        "allowedLevels": [
            "easy"
        ],
        "allowedMedia": [
            "image/png"
        ],
        "allowedInputType": [
            "text"
        ],
        "config": {}
    }, {
        "name": "RainDropsCaptcha",
        "allowedLevels": [
            "easy",
            "medium"
        ],
        "allowedMedia": [
            "image/gif"
        ],
        "allowedInputType": [
            "text"
        ],
        "config": {}
    }]
}
```

Configuration seems to be working fine.

However, even if the `playgroundEnabled` field is set to `false`, the demo is still served and available. I would like to disable this feature but I can't.

Thank you for any help

**Comment by hrj** [comment]
Thanks for the detailed report. There seems to be a bug [here](https://github.com/librecaptcha/lc-core/blob/master/src/main/scala/lc/core/models.scala#L39). A logical OR with `True` will always evaluate to `True`.

I will fix it soon and make a new release.

**Comment by hrj** [comment]
Hi @pitininja 

The bug is fixed and released as part of `2.0.1`.

Since you are using the `latest` image, it should work if you just update the image. Else, you can use the `2.0` or `2.0.1` tag.

**Comment by pitininja** [comment]
Thank you very much for the fix @hrj.

It works great now.
